src/main/resources/app.conf

```diff
@@ -0,0 +1,79 @@
+# All application specific configuration should reside here
+app {
+  # The service description namespace
+  description {
+    # The name of the service
+    name = "kg_v1"
+    environment = "local"
+    environment = ${?ENVIRONMENT}
+  }
+
+  # Service instance specific settings
+  instance {
+    # The default interface to bind to
+    interface = 127.0.0.1
+    interface = ${?BIND_INTERFACE}
+  }
+
+  # Http binding settings
+  http {
+    # The interface to bind to
+    interface = ${app.instance.interface}
+    # The port to bind to
+    port = 8080
+    port = ${?BIND_PORT}
+    # The default uri prefix
+    prefix = "v1"
+    prefix = ${?HTTP_PREFIX}
+    # The service public uri
+    public-uri = "http://"${app.http.interface}":"${app.http.port}
+    public-uri = ${?PUBLIC_URI}
+  }
+
+  # Service runtime settings
+  runtime {
+    # Arbitrary Future completion timeout
+    default-timeout = 30 seconds
+  }
+
+  # Cluster specific settings
+  cluster {
+    # Duration after which actors are shutdown if they have no interaction
+    passivation-timeout = 10 seconds
+    passivation-timeout = ${?PASSIVATION_TIMEOUT}
+    # Total number of shards in the cluster
+    shards = 100
```

Comment:
change the default to `30`; the recommendation is 10 times the number of nodes and we advise deployment to 3

Reply:
we should also add an env `SHARDS` to control this per deployment